Sources/SwiftUIBackports/Shared/Transition/PushTransition.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  PushTransition.swift
+//  
+//
+//  Created by Andrey Plotnikov on 10.08.2022.
+//
+
+import SwiftUI
+
+public extension AnyTransition {
+    static var backport: Backport<AnyTransition>{
+        Backport(.identity)
+    }
+}
+
+public extension Backport where Wrapped == AnyTransition {
+    /// Creates a transition that when added to a view will animate the view’s insertion by moving it in from the specified edge while fading it in, and animate its removal by moving it out towards the opposite edge and fading it out.
+    /// - Parameter edge: the edge from which the view will be animated in.
+    /// - Returns: A transition that animates a view by moving and fading it.
+    @available(iOS, deprecated: 16.0)
+    @available(watchOS, deprecated: 9.0)
+    @available(macOS, deprecated: 13.0)
+    @available(tvOS, deprecated: 16.0)
+    @available(macCatalyst, deprecated: 16.0)
```

Comment:
I don't have macCatalyst support anywhere else to simplify, so this isn't required
```suggestion
```